go.mod

```diff
@@ -33,6 +34,8 @@ require (
 	google.golang.org/protobuf v1.36.6
 )
 
+require github.com/ava-labs/coreth v0.15.1-rc.0.0.20250530184801-28421010abae
```

Comment:
Can we group this in the require block with the other ava-labs deps?

Reply:
`tests/abi-bindings/go/BatchCrossChainMessenger/BatchCrossChainMessenger.go` ends up with the direct dep somehow 👀